src/components/Books.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { removeBook } from '../redux/books/books';
+
+const Books = (props) => {
+  const {
+    book: {
+      books,
+      author,
+      id,
+      category,
+    },
+
+  } = props;
+  const dispatch = useDispatch();
+
+  return (
+    <>
+      <li key={id} className="books-content">
+        {books}
+        {' by '}
+        {author}
+        {' - '}
+        {category}
+      </li>
+      <button type="button" id={id} onClick={() => dispatch(removeBook(id))}>Remove</button>
```

Comment:
Kindly pass a named function to `onClick` event to removeBook. This is a project requirement.